content/elao/job-facilitateur.md

```diff
@@ -0,0 +1,64 @@
+---
+type:           "post"
+title:          "Facilitateur.ice projet, j'ai besoin d'aide."
+date:           "2017-02-01"
+publishdate:    "2017-02-01"
+draft:          true
+slug:           "job-facilitateur-agence-lyon"
+description:    "Nous recherchons un.e facilitateur.ice projet pour notre agence de Lyon."
+
+thumbnail:      "/images/posts/thumbnails/riccardo-chiarini-247364.jpg"
+header_img:     "/images/posts/headers/riccardo-chiarini-247364.jpg"
+tags:           ["agile", "jobs", "lyon", "gestion de projet", "facilitateur"]
+categories:     ["jobs"]
+
+author_username:    "xgorse"
+---
+
+<!--more-->
+## Un.e facilitateur.ice projet
+
+Elao est un **atelier de développement logiciel** qui propose à ses clients la conception et le développement d’applications Web/Mobile métier **sur mesure**. Nous apprécions particulièrement les projets de 3 à 6 mois où l'on peut mettre une équipe de 3 personnes dédiée au projet.
+
+Notre **promesse** est de chercher à fournir les solutions techniques et une organisation projet les plus **efficaces** pour délivrer un maximum de **valeur ajoutée** aux produits que l'on développe pour nos clients.
+Chaque projet comme chaque client est **unique**, nous sommes donc en perpétuelle **expérimentation** dans cette quête de l'efficacité.
+
+Votre rôle sera d'épauler aussi bien l'équipe de dev que les clients dans cette quête:
+
+ * En challengant avec l'équipe les besoins client en amont, en **l'aidant à prioriser** ses attentes, ou bien en le **rassurant** et en lui tenant la main dans sa découverte de la réalisation de projet en mode itératif.
```

Comment:
> en lui tenant la main

L'expression fait un peu infantilisante je trouve.

Reply:
...[et en lui permettant de comprendre simplement les différentes étapes et itérations de réalisation du projet.]